docs/staking_claim_rewards.md

```diff
@@ -0,0 +1,20 @@
+---
+id: staking_claim_rewards
+title: Claim your Staking Rewards
```

Comment:
Need to set a standard for headings now. 
I prefer capitalized headings but the rest of the titles must be changed too then.